config/helm/chart/default/values.yaml

```diff
@@ -220,7 +220,4 @@ rbac:
   kspm:
     create: true
     annotations: {}
-  kubernetesMonitoringFull:
-    create: false
-    annotations: {}
```

Comment:
just to clarify, we don't want the `preview` section in the `values.yaml` for a reason, right?